src/app-prisma/prisma.service.ts

```diff
@@ -0,0 +1,15 @@
+import { INestApplication, Injectable, OnModuleInit } from '@nestjs/common';
+import { PrismaClient } from '@prisma/client';
+
+@Injectable()
+export class PrismaService extends PrismaClient implements OnModuleInit {
```

Comment:
For example

```
constructor(private configService: ConfigService<EnvironmentVariables>) {
    /**
     * Get the database url from environmental variables and pass it in.
     */
    super({
      datasources: {
        db: {
          // url: configService.get('DATABASE_URL'),
          name: configService.get('DB_NAME')
        },
      },
    });
  }
```

Reply:
prisma service has no interaction with prisma schema.

Reply:
@MoutasemHabboub https://dev.to/harryhorton/how-to-use-prisma-with-nestjs-3l1b